data.sql

```diff
@@ -44,3 +44,26 @@ VALUES ('Blossom', '1998-10-13', 3, true, 17);
 INSERT INTO animals 
 (name, date_of_birth, escape_attempts, neutered, weight_kg) 
 VALUES ('Ditto', '2022-05-14', 4, true, 22);
+
+-- query multiple updates
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34);
+INSERT INTO owners (full_name, age) VALUES ('Jennifer Orwell', 19);
+INSERT INTO owners (full_name, age) VALUES ('Bob', 45);
+INSERT INTO owners (full_name, age) VALUES ('Melody Pond', 77);
+INSERT INTO owners (full_name, age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (full_name, age) VALUES ('Jodie Whittaker', 38);
+
+INSERT INTO species (name) VALUES ('Pokemon');
+INSERT INTO species (name) VALUES ('Digimon');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species_id, owner_id) VALUES
+    ('Agumon', '2018-05-10', 2, true, 15.2, 2, 1),
+    ('Gabumon', '2019-02-15', 0, false, 12.6, 2, 2),
+    ('Pikachu', '2020-08-03', 1, false, 5.7, 1, 2),
+    ('Devimon', '2017-11-21', 3, false, 20.1, 2, 3),
+    ('Plantmon', '2018-09-28', 0, true, 8.5, 2, 3),
+    ('Charmander', '2016-06-12', 0, true, 9.9, 1, 4),
+    ('Squirtle', '2016-09-17', 0, true, 7.3, 1, 4),
+    ('Blossom', '2017-03-04', 0, true, 6.1, 1, 4),
+    ('Angemon', '2019-12-09', 2, true, 18.7, 2, 5),
+    ('Boarmon', '2020-05-27', 0, false, 14.3, 2, 5);
```

Comment:
![image](https://github.com/mahabubx7/vet_clinic/assets/93039359/cbb3c72b-c639-420e-b634-c0e0e5af6e09)

- [ ] It would be nice if you follow the project requirements then you would find out that you are supposed to update your tables data instead of inserting the new data 🤔 Because it is clearly mentioned that `modify your inserted animals` so it would be great if you use **Update** query here to follow the project requirements 👍